### PR TITLE
Citesphere app page

### PR DESCRIPTION
**Pull request by adhvikmadhav**

### CODE REVIEW

**Comment by jdamerow** [comment]
- in the dropdown menu "our work", none of the app links work
- let's keep the github commit activity, maybe put it below FAQs
- there are issues with the dark mode
- also the dark mode is not kept anymore, when I go to another page it's in the light mode again
- on the Citesphere page, let's put the "Sync your Zotero citations." sections first.

please reopen this PR next time, so the comments are kept.

**Comment by jdamerow** [comment]
also, let's add the link to the citephere documentation (https://diging.atlassian.net/wiki/spaces/CITED/overview).

**Comment by adhvikmadhav** [comment]
Review

File: _includes/asu_navigation.html [hunk 1]
```diff
@@ -53,12 +53,12 @@
...
-        {% endfor %}
-        {% for items in site.data.mini-projects %}
-          $("#{{ items.id }}").attr('src',"{{ items.image-color }}")
-        {% endfor %}
+        // {% for item in site.data.projects %}
+        //   $("#{{ item.id }}").attr('src',"{{ item.image-color }}")
+        // {% endfor %}
+        // {% for items in site.data.mini-projects %}
+        //   $("#{{ items.id }}").attr('src',"{{ items.image-color }}")
+        // {% endfor %}
```
Comment (jdamerow) [inline _includes/asu_navigation.html:61]: remove if not needed

File: projects/index2.html [hunk 2]
```diff
@@ -0,0 +1,319 @@
...
+														<h3>Black Goat</h3>
+													</header>
+													<p>The primary objective of this project is provide a platform for non-hierarchical alignments among an unlimited number of authority systems. This is primarily intended for projects that introduce a significant number of concepts in their own namespace, a substantial proportion of which overlap with other authority systems.</p>
+													<footer>
+														<a href="https://diging.github.io/black-goat/" class="button alt"><span class="icon fa-github"></span> Read more on GitHub.io</a>
+													</footer>
+												</section>
+											  </div>
+                      	</div>
+										</div> -->
```
Comment (jdamerow) [inline projects/index2.html:147]: remove if no longer needed

File: projects/index2.html [hunk 3]
```diff
@@ -0,0 +1,319 @@
...
+  														<h3>History of Developmental Evolution</h3>
+  													</header>
+  													<p>Developmental Evolution is a tradition within evolutionary biology with a very long history. It focuses on the mechanistic (cellular, developmental, genomic) causes that generate phenotypes and phenotypic variation. While it is often seen as an "alternative" to mainstream population genetics, it is in fact a complementary and necessary approach to theories of evolutionary dynamics. This project looks at the history of this tradition as a big data problem, combining the reconstruction of individual contributions and research programs with computational analysis of the conceptual dynamics of evolutionary biology at large.</p>
+  													<footer>
+  														<a href="http://devo-evo.lab.asu.edu/history-developmental-evolution" class="button alt">Find out more</a>
+  													</footer>
+  												</section>
+  											</div>
+                        </div>
+										</div> -->
```
Comment (jdamerow) [inline projects/index2.html:203]: same as above (remove if not needed)

**Review by jdamerow — CHANGES_REQUESTED** [review]
The links to the Citesphere page don't work.

File: _data/applications.yml [hunk 4]
```diff
@@ -0,0 +1,42 @@
+
+- image-color: projects/citesphere-logo.png
+  image-white: projects/citesphere-white.png
+  url: ./projects/app-pages/citesphere
+  id: citesphere-img-1
+  name: Citesphere
+  description: "Manage your citations and more."
```
Comment (jdamerow) [inline _data/applications.yml:7]: Let's make this "Manage your references and more."

File: _data/applications.yml [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+  id: conceptpower-img-1
+  name: Conceptpower
+  description: "Search for concepts and related words."
+
+- image-color: projects/vspace-logo.png
+  image-white: projects/virtualspaces-white.png
+  url: https://github.com/diging/virtual-spaces-2.0
+  id: virtualspaces-img-1
+  name: Virtual Spaces 2.0
+  description: "Build virtual exhibitions through 2D space."
```
Comment (jdamerow) [inline _data/applications.yml:21]: "Build virtual exhibitions online."

File: _data/projects.yml [hunk 6]
```diff
@@ -1,47 +1,7 @@
+- name: Women in Computer Science
+  desc: "Network map of remarkable women engineers."
```
Comment (jdamerow) [inline _data/projects.yml:2]: Network map of remarkable women in computer science.

File: _data/tools.yml [hunk 7]
```diff
@@ -0,0 +1,20 @@
+
+- name: DigInG Icon Pack
+  desc: "Web icons made for commercial or personal use."
+  url: https://github.com/diging/diging-icon-pack
+
+- name: JSTOR Importer
+  desc: "Import JSTOR data for corpus analysis into Elasticsearch."
```
Comment (jdamerow) [inline _data/tools.yml:7]: Import your JSTOR data into Elasticsearch.

**Review by jdamerow — CHANGES_REQUESTED** [review]

**Comment by jdamerow** [comment]
The link to citesphere on the "Our work" page doesn't work.

**Pull request merged**